bundles/org.openhab.persistence.jpa/src/main/resources/OH-INF/i18n/jpa.properties

```diff
@@ -1,3 +1,17 @@
+# add-on
+
+addon.jpa.name = JPA Persistence
+addon.jpa.description = This is the persistence add-on for JPA.
+
+# add-on config
+
+addon.config.jpa.driver.label = JDBC Driver
+addon.config.jpa.password.label = Password
+addon.config.jpa.url.label = JDBC connection URL
+addon.config.jpa.user.label = User
+
+# add-on config
```

Comment:
Duplicated header - I'm wondering if the lines below need to be tweaked as well (persistence -> addon)? The new ones above also exist in other variations below.

Reply:
I think we should use `addon.` and remove `service.`

When the tool generated translations for persistence add-ons without i18n files, `addon.` was used - `service.` was not used.

Reply:
This still looks odd: Are these different texts shown in different places?
```ini
# add-on config

addon.config.jpa.driver.label = JDBC Driver
addon.config.jpa.password.label = Password
addon.config.jpa.url.label = JDBC connection URL
addon.config.jpa.user.label = User
```

and (below):
```ini
# add-on config

persistence.config.jpa.driver.label = Database Driver
persistence.config.jpa.password.label = Database Password
persistence.config.jpa.url.label = Database URL
persistence.config.jpa.user.label = Database User
```

?